Update configuration.md

sebastienros opened the pull request:
Without a specific package the sample doesn't compile. The solution is only found in one of the comments. Cost me one hour too so I assume it won't be the last time hits this.

dnfclas commented:
@sebastienros,
Thanks for having already signed the Contribution License Agreement. Your agreement was validated by .NET Foundation. We will now review your pull request.
_Thanks,
.NET Foundation Pull Request Bot_

Rick-Anderson commented:
@sebastienros  sorry about that. Removed obtrusive **NOTE** and rewrote as 
Note: This extension method requires the `Microsoft.Extensions.Options.ConfigurationExtensions` NuGet package.

sebastienros commented:
Nothing to be sorry about. Thank you for keeping it consistent.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________
From: Rick Anderson <notifications@github.com>
Sent: Saturday, May 6, 2017 11:43:45 AM
To: aspnet/Docs
Cc: Sebastien Ros; Mention
Subject: Re: [aspnet/Docs] Update configuration.md (#3307)


@sebastienros<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsebastienros&data=02%7C01%7Csebastien.ros%40microsoft.com%7C49e44530e0cf4952bbf108d494afd4e7%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636296930298166836&sdata=BtF%2FOjbCd72P1oKX0Q6Hr7lKLcALUvC%2B6T%2BWRF1ZJJc%3D&reserved=0> sorry about that. Removed obtrusive NOTE and rewrote as
Note: This extension method requires the Microsoft.Extensions.Options.ConfigurationExtensions NuGet package.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Faspnet%2FDocs%2Fpull%2F3307%23issuecomment-299658564&data=02%7C01%7Csebastien.ros%40microsoft.com%7C49e44530e0cf4952bbf108d494afd4e7%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636296930298176849&sdata=s5OaLb13eQRBO5LcuMTHMKlHVV5L95UXuhoNVf%2BXee4%3D&reserved=0>, or mute the thread<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FABHJ7abgjXWZGULp1uzKO9A_VJH1f_gCks5r3L9hgaJpZM4NRDhM&data=02%7C01%7Csebastien.ros%40microsoft.com%7C49e44530e0cf4952bbf108d494afd4e7%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636296930298176849&sdata=Q6wkegO7uR%2FwRPyKZa4LE%2FlvMBlgag%2FDXKZAgFTrfYg%3D&reserved=0>.
